about.html

```diff
@@ -0,0 +1,301 @@
+                        </li>
+                        <li>
+                            <a class="follow" href="##">Follow us</a>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+        </nav>
+
+        <!--main section-->
+        <div class="container-fluid main-section">
+
+
+            <div class="container hambuger" id="hambuger_about">
+                <div class="row">
+                    <div class="col-12 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
+
+                <!--Hamburger drop down-->
+                <div class="container-fluid hide-menu" id="hide_menu_about">
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+
+                            <ul>
+                                <li>
+                                    <a href="index.html">Home</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="closeButton" id="close_button_about">&times;</span>
+                        </div>
+                    </div>
+                </div>
+
+
+            </div>
+            <div class="row">
+
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="col-3 desk-div"></div>
+                <div class="col-9">
+                    <h1 class="about_header">RR Conference 2023.
+                        <br> <span class="about_titlespan"></span>
+                    </h1>
+                </div>
+            </div>
+            <div class="row">
+
+                <div class="col-12 about_paragraph_second">
+
+
+                    <p> In particular, this year's summit is our third since incession
+                        It's necessary when you have a company larger than 50 people,
+                        when you need to share vision and values with the whole company
+                        the best way to do it is in a conference format. This is also a
+                        great format for teaching and sharing learnings from projects and people management.
```

Comment:
- [ ] Kindly make sure all the content of the container section are center aligned just according to the design guidelines. 

Design Guidelines:

<img width="487" alt="Screenshot 2023-03-24 at 11 10 13 PM" src="https://user-images.githubusercontent.com/44798044/227617110-98b434a0-f1e4-48cc-aa77-ca5c53eaa7dc.png">

Your design:


<img width="378" alt="Screenshot 2023-03-24 at 11 08 34 PM" src="https://user-images.githubusercontent.com/44798044/227617229-cafa171e-7528-4928-bf1e-4d831156adc8.png">